examples/stats/basic_prototype.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import monitoring
+from google.cloud.monitoring import MetricDescriptor
+from opencensus.stats.exporters import stackdriver_exporter
+from opencensus.stats import measure
+from opencensus.stats import view
+from opencensus.stats import aggregation
+
+def main():
+    views = []
+
+    client = monitoring.Client(project='opencensus-python')
+    exporter = stackdriver_exporter.StackDriverExporter(client, 'opencensus-python')
```

Comment:
Exporter should accepts a project id and create the client in constructor.

Reply:
I'm pretty sure that is covered in the __init__ function of the exporter, is it not?